### PR TITLE
feat: rotate cesium map instead of infinite scroll and adding it to t…

### PR DESCRIPTION
**Pull request by Eliav-Maimon**
…he env

### CODE REVIEW

File: src/discrete-layer/views/discrete-layer-view.tsx [hunk 1]
```diff
@@ -100,6 +101,8 @@ const DELTA = 0.00001;
 const DRAWING_MATERIAL_OPACITY = 0.5;
 const DRAWING_FINAL_MATERIAL_OPACITY = 0.8;
 const DRAWING_MATERIAL_COLOR = CesiumColor.YELLOW.withAlpha(DRAWING_MATERIAL_OPACITY);
+const mapMode2D = MapMode2D[CONFIG.MAP.MAP_MODE_2D];
```
Comment (alebinson) [inline src/discrete-layer/views/discrete-layer-view.tsx:104]: think conversion should be more watter-proof. If value NOT on of ROTATE/INFINIT_SCROLL then INFINIT_SCROLL should be resolved

**Review by alebinson — CHANGES_REQUESTED** [review]
see my comments

File: src/discrete-layer/views/discrete-layer-view.tsx (hunk 1 above)
Comment (Eliav-Maimon) [reply src/discrete-layer/views/discrete-layer-view.tsx:104]: When in config it's not a valid option, Cesium's default will apply with the value of INFINITE_SCROLL.

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**